src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt

```diff
@@ -25,6 +39,20 @@ class NomisMappingClient(private val nomisMappingApiWebClient: WebClient) {
     .block()
 }
 
+class DpsLocationsIds(locationIds: Collection<UUID>) {
```

Comment:
Wrapping the IDs in a class means we can better support caching to ensure the order of the IDs are always in the same order.  If the same IDs were used but in a different order then it would be seen as an entirely new request.